themes/default/content/blog/docker-build/index.md

```diff
@@ -0,0 +1,744 @@
+# you want to be able to merge the post without publishing it.
+draft: false
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or
+# social-media previews. This field is required or the build will fail the
+# linter test. Max length is 160 characters.
+meta_desc: >-
+    Pulumi's new Docker Build provider offers support for Docker builds using modern buildx/BuildKit APIs.
+
+# The meta_image appears in social-media previews and on the blog home page. A
+# placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the
+# `id` properties of the team member files at /data/team/team. Create a file for
+# yourself if you don't already have one.
+authors:
+    - bryce-lampe
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - docker
+    - buildx
+    - buildkit
+    - containers
+
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md
+# for details, and please remove these comments before submitting for review.
+---
+
+We are thrilled to introduce the latest addition to the Pulumi Ecosystem -- the new _Docker-Build_ provider, designed to streamline and enhance Docker image builds.
+This addition builds on the solid foundation of our widely utilized [Pulumi Docker] provider.
+By embracing the power of Docker's [BuildKit] technology, this provider enhances your building experience, allowing you to modernize your image builds.
+This launch signifies our ongoing commitment to advancing technology and empowering developers to achieve new heights in container management.
+
+The Docker-Build provider is crafted to optimize the efficiency and versatility of Docker builds and offers developers a more potent and flexible approach to building Docker images, ensuring that they can achieve high performance and scalability in their deployment workflows.
+The new provider exposes the full power of Docker's next-gen [buildx](https://docs.docker.com/reference/cli/docker/buildx/build/) functionality and includes key features such as,
+
+* __Multi-platform image support__: Build images that run seamlessly across different hardware architectures.
+
+* __Advanced caching mechanisms__: Utilize various caching strategies to speed up builds and reduce resource consumption.
+
+* __Support for secrets and Pulumi [ESC](../environments-secrets-configurations-management) (Environments, Secrets and Configuration)__: Safely manage sensitive information and configuration with first-class support, integrating seamlessly into your deployment workflows.
+
+* __Support for multiple export types__: Increase the power and flexibility of your workflows by exporting your images to registries, disk, or blob storage.
```

Comment:
I don't see any later mention of this.  Worth adding into an example or providing more details elsewhere in the post?